src/stripeTreeItem.ts

```diff
@@ -1,18 +1,25 @@
 import {TreeItem, TreeItemCollapsibleState} from 'vscode';
 
+type StripeTreeItemOptions = Pick<TreeItem, 'contextValue' | 'tooltip'> & {
```

Comment:
I picked only the class members from `vscode.TreeItem` we were using, then extended it with `commandString` because it's `StripeTreeItem`'s custom parameter.

Reply:
Nice!